include/rnp/rnp2.h

```diff
@@ -260,6 +264,54 @@ rnp_result_t rnp_key_have_public(rnp_key_handle_t key, bool *result);
 
 /* TODO define functions for password-based encryption */
 
+rnp_result_t rnp_op_sign_create(rnp_op_sign_t *op,
+                                rnp_ffi_t      ffi,
+                                rnp_input_t    input,
+                                rnp_output_t   output);
+
+rnp_result_t rnp_op_sign_set_signing_key(rnp_op_sign_t op, rnp_key_handle_t key);
```

Comment:
I think this should be named `rnp_op_sign_add_signer` to align with `rnp_op_encrypt_add_recipient`.

Reply:
Agree, it will also go well with multiple signers.

Reply:
Fixed